array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,14 @@
+/* exported equal */
+function equal(first, second) {
+  var combinedArr = [];
+  for (var i = 0; i < first.length; i++) {
+    if (first[i] === second[i]) {
+      combinedArr.push(first[i]);
+    }
```

Comment:
You can change this to check for not equal (`!==`) if that is true you can just return false from the function. You don't need to accumulate all the matches.